Flaky verify: pull-kueue-verify-main fails occassionally

<!--
Please use this template for reporting flaky tests.
Links to specific failures in Prow are appreciated.
-->

**Which test is flaking?**:
pull-kueue-verify-main 

**Link to failed CI job or steps to reproduce locally**:
https://storage.googleapis.com/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_kueue/10999/pull-kueue-verify-main/2052265488649555968/build-log.txt

**Failure message or logs**:
```
0 issues.
make[1]: Leaving directory '/home/prow/go/src/sigs.k8s.io/kueue'
git --no-pager diff --exit-code config/components apis charts/kueue client-go keps site/ netlify.toml internal/mocks
diff --git a/site/content/en/docs/reference/kueue-config.v1beta1.md b/site/content/en/docs/reference/kueue-config.v1beta1.md
index 3c8d8abfc..f18f22a4f 100644
--- a/site/content/en/docs/reference/kueue-config.v1beta1.md
+++ b/site/content/en/docs/reference/kueue-config.v1beta1.md
@@ -58,7 +58,7 @@ unsuspended, they will start immediately.</p>
 </td>
 </tr>
 <tr><td><code>managedJobsNamespaceSelector</code> <B>[Required]</B><br/>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#labelselector-v1-meta"><code>k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector</code></a>
+<code>invalid type</code>
 </td>
 <td>
    <p>ManagedJobsNamespaceSelector provides a namespace-based mechanism to exempt jobs
@@ -184,7 +184,7 @@ of Kueue-managed objects. A nil value disables all automatic deletions.</p>
     
   
 <tr><td><code>usageHalfLifeTime</code> <B>[Required]</B><br/>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#duration-v1-meta"><code>k8s.io/apimachinery/pkg/apis/meta/v1.Duration</code></a>
+<code>invalid type</code>
 </td>
 <td>
    <p>usageHalfLifeTime indicates the time after which the current usage will decay by a half
@@ -192,7 +192,7 @@ If set to 0, usage will be reset to 0 immediately.</p>
 </td>
 </tr>
 <tr><td><code>usageSamplingInterval</code> <B>[Required]</B><br/>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#duration-v1-meta"><code>k8s.io/apimachinery/pkg/apis/meta/v1.Duration</code></a>
+<code>invalid type</code>
 </td>
 <td>
    <p>usageSamplingInterval indicates how often Kueue updates consumedResources in FairSharingStatus
@@ -822,7 +822,7 @@ If left empty, then metrics will expose for all local queues across namespaces.<
 </td>
 </tr>
 <tr><td><code>localQueueSelector</code><br/>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#labelselector-v1-meta"><code>k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector</code></a>
+<code>invalid type</code>
 </td>
 <td>
    <p>LocalQueueSelector can be used to choose the local queues that need metrics to be collected.</p>
@@ -846,7 +846,7 @@ If left empty, then metrics will expose for all local queues across namespaces.<
     
   
 <tr><td><code>gcInterval</code><br/>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#duration-v1-meta"><code>k8s.io/apimachinery/pkg/apis/meta/v1.Duration</code></a>
+<code>invalid type</code>
 </td>
 <td>
    <p>GCInterval defines the time interval between two consecutive garbage collection runs.
@@ -865,7 +865,7 @@ them if their local counterpart no longer exists.</p>
 </td>
 </tr>
 <tr><td><code>workerLostTimeout</code><br/>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#duration-v1-meta"><code>k8s.io/apimachinery/pkg/apis/meta/v1.Duration</code></a>
+<code>invalid type</code>
 </td>
 <td>
    <p>WorkerLostTimeout defines the time a local workload's multikueue admission check state is kept Ready
@@ -966,14 +966,14 @@ A nil value disables automatic deletion of Workloads.</p>
     
   
 <tr><td><code>namespaceSelector</code> <B>[Required]</B><br/>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#labelselector-v1-meta"><code>k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector</code></a>
+<code>invalid type</code>
 </td>
 <td>
    <p>NamespaceSelector can be used to omit some namespaces from pod reconciliation</p>
 </td>
 </tr>
 <tr><td><code>podSelector</code> <B>[Required]</B><br/>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#labelselector-v1-meta"><code>k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector</code></a>
+<code>invalid type</code>
 </td>
 <td>
    <p>PodSelector can be used to choose what pods to reconcile</p>
@@ -1273,7 +1273,7 @@ Defaults to false.</p>
 </td>
 </tr>
 <tr><td><code>timeout</code><br/>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#duration-v1-meta"><code>k8s.io/apimachinery/pkg/apis/meta/v1.Duration</code></a>
+<code>invalid type</code>
 </td>
 <td>
    <p>Timeout defines the time for an admitted workload to reach the
@@ -1299,7 +1299,7 @@ This setting is only honored when <code>Enable</code> is set to true.</p>
 </td>
 </tr>
 <tr><td><code>recoveryTimeout</code><br/>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#duration-v1-meta"><code>k8s.io/apimachinery/pkg/apis/meta/v1.Duration</code></a>
+<code>invalid type</code>
 </td>
 <td>
    <p>RecoveryTimeout defines a timeout, measured since the
@@ -1331,7 +1331,7 @@ Defaults to the value of timeout. Setting to &quot;0s&quot; disables recovery ti
     
   
 <tr><td><code>afterFinished</code><br/>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#duration-v1-meta"><code>k8s.io/apimachinery/pkg/apis/meta/v1.Duration</code></a>
+<code>invalid type</code>
 </td>
 <td>
    <p>AfterFinished is the duration to wait after a Workload finishes
@@ -1342,7 +1342,7 @@ Represented using metav1.Duration (e.g. &quot;10m&quot;, &quot;1h30m&quot;).</p>
 </td>
 </tr>
 <tr><td><code>afterDeactivatedByKueue</code><br/>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#duration-v1-meta"><code>k8s.io/apimachinery/pkg/apis/meta/v1.Duration</code></a>
+<code>invalid type</code>
 </td>
 <td>
    <p>AfterDeactivatedByKueue is the duration to wait after <em>any</em> Kueue-managed Workload
make: *** [Makefile-verify.mk:65: verify] Error 1
+ EXIT_VALUE=2
+ set +o xtrace
```

**Anything else we need to know?**:

--- a/Makefile
+++ b/Makefile
@@ -471,7 +471,7 @@ kueuectl:
 	CGO_ENABLED=$(CGO_ENABLED) $(GO_BUILD_ENV) $(GO_CMD) build -ldflags="$(LD_FLAGS)" -o $(BIN_DIR)/kubectl-kueue cmd/kueuectl/main.go
 
 .PHONY: generate-apiref
-generate-apiref: genref
+generate-apiref: genref generate-code
 	cd $(PROJECT_DIR)/hack/genref/ && $(GENREF) -o $(PROJECT_DIR)/site/content/en/docs/reference
 
 ##@ Documentation

